docs/self-managed/zeebe-deployment/zeebe-gateway/job-streaming.md

```diff
@@ -0,0 +1,22 @@
+---
+id: job-streaming
+title: "Job Streaming"
+sidebar_label: "Job Streaming"
+---
+
+Streaming job workers are expected to be long lived in order to cut down on the latency overhead involved with (re)creating a stream and propagating this throughout the cluster. This may require special configuration, especially if you're using a reverse proxy in front of your gateway. Typically, this will affect you in the form of HTTP 504 (Gateway Timeout) being returned to your job streaming worker at regular intervals.
+
+:::note
+Note that this configuration is _only_ required for reverse proxies which do not support forwarding HTTP/2 keepalive (on either side). See, for example, [this nginx ticket](https://trac.nginx.org/nginx/ticket/1887).
+
+If your proxy supports it, then you don't need to do anything.
+:::
+
+The general recommendation would be to apply the following configuration:
+
+- On your client, set an explicit stream timeout, say, 1h.
```

Comment:
:wrench: Mention or link to the config for that

Reply:
It's really client specific though, isn't it? :thinking: I could link to each official client I suppose.

Reply:
I'd link to the java client just to have an example.